codes/qubits/steane.yml

```diff
@@ -0,0 +1,60 @@
+#######################################################
```

Comment:
@jtiosue Forgot to add this... can u tack it on to realizations:

- 'Realized in trapped-ion devices \cite{arXiv:2107.07505}.'

Reply:
@valbert4 Isn't that already there, in line 44? Am I missing something?

Reply:
Sorry, you're right @jtiosue. Add this one as well: arXiv:1403.5426